packages/eds-core-react/src/components/Tabs/Tabs.docs.mdx

```diff
@@ -0,0 +1,122 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Tabs } from '.'
+
+# Tabs
+
+Organise related content across different views to be quickly navigated.
+
+<Links 
+    ariaUrl="https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/"
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=17417%3A0"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/50d3be-tabs/b/94939a"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Tabs/Tabs.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="navigation-tabs--introduction" />
+<PropsTable />
+
+## Usage
+
+Tabs allow for navigation between content that is related and has an equal hierarchy.
+
+<ul>
+    <li>Each tab's content should be noticeably different from the other.</li>
+    <li>Tabs allow for information to be presented in a space-saving layout.</li>
+    <li>Tab labels should provide clear and concise direction.</li>
+    <li>Note that if the label is longer than the maximum width it will be truncated and should have a tooltip provided.</li>
+</ul>
+
+```tsx
+import { Dialog } from '@equinor/eds-core-react'
+
+<Tabs>
+    <Tabs.List>
+      <Tabs.Tab>One</Tabs.Tab>
+      <Tabs.Tab>Two</Tabs.Tab>
+    </Tabs.List>
+    <Tabs.Panels>
+      <Tabs.Panel>Panel one</Tabs.Panel>
+      <Tabs.Panel>Panel two</Tabs.Panel>
+    </Tabs.Panels>
+</Tabs>
+
+```
+
+## Examples
+
+### States
+
+<Story id="navigation-tabs--states" />
+
+### Widths
+
+Tabs have:
+<ul>
+    <li><code>minWidth</code> width of Spacer/16 - Medium on each side of the label.</li>
```

Comment:
It is confusing to me what "Spacer/16 - Medium" means

Reply:
Yes it is 😏